src/app/component/about/components/about-page/about-page.component.html

```diff
@@ -1,45 +1,46 @@
-<div class="page-wrapper">
-  <div class="container-about">
-    <div class="full-text-block">
-      <h2>
-        {{ "about-us.intro.block-1.header" | translate }}
-      </h2>
-      <p>
-        {{ 'about-us.intro.block-1.text' | translate }}
-      </p>
-      <button class="primary-global-button" (click)="navigateToHabit()">
-        {{ 'about-us.intro.block-1.button' | translate }}
-      </button>
-    </div>
-    <img src="assets/img/Illustrationman.png" alt="Illustration man">
-  </div>
-
-  <div class="vision">
-    <div class="container-vision">
-      <img src="assets/img/Vision.png" alt="vision">
+<main id="main-content" role="main">
+  <div class="page-wrapper">
+    <div class="container-about">
       <div class="full-text-block">
         <h2>
-          {{ 'about-us.intro.block-2.header' | translate }}
+          {{ "about-us.intro.block-1.header" | translate }}
```

Comment:
Why did you use double quotes in this row only? We use single quotes in interpolation.

Reply:
Actually I didn't.. these are not my changes.